easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-open-4.0.6-foss-2016a-Perl-5.22.1.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PackedBinary'
+
+name = 'RepeatMasker'
+version = 'open-4-0-6'
```

Comment:
@RvDijk I would include the `open` in the `sources` only, not in the actual version

And I would use `version = '4.0.6'` here, and transform that into `4_0_6` in `sources` via `'_'.join(version.split('.'))`